### DISCUSSION TITLE
feature: add a shortcut option for actually taking a screenshot

### DISCUSSION THREAD

**Discussion by caveyspider** [flameshot-org/flameshot#3280] [Q&A] [answered]
before you read the next part of this i want you to know i know **NOTHING** about coding or stuff

this is a thing that's totally about preferences, but as someone who was very used to taking screenshots with windows + shift + s and now i can't use it,  this just seems like the most sensible option for customization, if you can actually do that and there's not some weird privacy thing involved that is

**Comment by mmahmoudian** [comment]
You did not provide enough information for us to help you. What operating system so you use, what is your Flameshot version?

**Reply by caveyspider** [reply]
i'm using pop_OS 22.04 LTS and am on flameshot version  12.1.0

**Reply by mmahmoudian** [reply, marked as answer]
It is always nice to help a fellow Linux user :) I made a quick video to demo how to set it up on your machine.

If you have not installed the Flatpak version (PopOS by default uses the Flatpak, but you can choose from the menu to use "deb" as well), then instead of `flatpak run org.flameshot.Flameshot gui`, you can directly use `flameshot gui`.

Basically in this video I'm showing that you can bind a command to a keybinding.

[Peek 2023-08-03 18-12.webm](https://github.com/flameshot-org/flameshot/assets/390889/ed10af28-4849-4371-8297-2d09d21c4cc7)

**Comment by caveyspider** [comment]
![image](https://github.com/flameshot-org/flameshot/assets/140642187/025f2bcb-faf3-445c-80a6-7c201cfbdf54)
thank you so much!!!!!!!
